avm/res/db-for-postgre-sql/flexible-server/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -62,62 +62,71 @@ module diagnosticDependencies '../../../../../../utilities/e2e-template-assets/t
 // ============== //
 
 @batchSize(1)
-module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem' ]: {
-  scope: resourceGroup
-  name: '${uniqueString(deployment().name, resourceLocation)}-test-${serviceShort}-${iteration}'
-  params: {
-    name: '${namePrefix}${serviceShort}001'
-    location: resourceLocation
-    administrators: [
-      {
-        objectId: nestedDependencies.outputs.managedIdentityClientId
-        principalName: nestedDependencies.outputs.managedIdentityName
-        principalType: 'ServicePrincipal'
-      }
-    ]
-    skuName: 'Standard_D2s_v3'
-    tier: 'GeneralPurpose'
-    configurations: [
-      {
-        name: 'log_min_messages'
-        source: 'user-override'
-        value: 'INFO'
-      }
-      {
-        name: 'autovacuum_naptime'
-        source: 'user-override'
-        value: '80'
+module testDeployment '../../../main.bicep' = [
```

Comment:
Now that you added private endpoints and private endpoints being one of the poster-childs for WAF-alignment it might be a good call to add the private endpoint deployment to this test too. 
Again, it would show how to deploy the module in a specific way (/scenario), this time being in a WAF-aligned one 💪

Reply:
This is not possible, in the WAF test postgres is created in a delegated subnet with 'private access' which disables the option to create own private endpoints, hence the seperate test.